src/styles/Styles.js

```diff
@@ -1,7 +1,7 @@
 import styled from 'styled-components';
 
 const Container = styled.div`
-    background-color: #ffffff;
+    background-color: ${props => props.theme};
```

Comment:
Did you miss the variable from theme here ? Becuase from what I understood theme is only a boolean ?

Reply:
wow didnt notice this, good catch